### ISSUE TITLE
Class overrides a field of a parent class - false-positive?

### ISSUE THREAD

**Issue by JackWH** [kalessil/phpinspectionsea#365]
I'm writing a Laravel application, which creates its models by extending the `Model` class:

```
abstract class Model
{
    /**
     * The table associated with the model.
     * 
     * @var string
     */
    protected $table;

    ...
}
```

My class looks like this:

```
class MyModel extends Model
{
    /**
     * The table associated with the model.
     * 
     * @var string
     */
    protected $table = 'my_table_name';

    ...
}
```

I'm being warned that *'Field "table" is already defined'*, as part of the [Class overrides field of parent class](https://github.com/kalessil/phpinspectionsea/blob/master/docs/architecture.md#class-overrides-a-field-of-a-parent-class) inspection.

But I don't understand why the warning is triggering in this case. I'm simply setting a default value on the property. The property has the same visibility as the parent class. I get the issues if you're overriding it with a different visibility, but that's not the case here. The inspection doesn't warn if I use `public $table = 'my_table_name';` either.

I might be missing something, but what is 'wrong' with doing this? Thanks in advance.

**Comment by kalessil** [comment]
Hi @JackWH,

It looks like you are forced to follow this approach, so lets phrase t like "nothing 'wrong' in terms of the used framework".

In general the inspection points to the Model class design issues - the class needs proper constructor like `public function __construct(string $table, array $attributes = array())` or another builders approach maybe.
